lib/tessel.js

```diff
@@ -6,6 +6,7 @@ var tessel;
 var factory;
 
 if (process.env.IS_TEST_ENV) {
+  __dirname = "/app/remote-script";
```

Comment:
If you make a function outside of this condition, that can both get and set the __dirname, the expose that function in here, you can change the value from your tests